core/server/master/src/main/java/alluxio/master/meta/DefaultMetaMaster.java

```diff
@@ -731,9 +731,22 @@ public Map<String, Boolean> updateConfiguration(Map<String, String> propertiesMa
         PropertyKey key = PropertyKey.fromString(entry.getKey());
         if (Configuration.getBoolean(PropertyKey.CONF_DYNAMIC_UPDATE_ENABLED)
             && key.isDynamic()) {
-          Object oldValue = Configuration.get(key);
-          Object value = key.parseValue(entry.getValue());
-          Configuration.set(key, value, Source.RUNTIME);
+          Object oldValue = null;
+          try {
+            oldValue = Configuration.get(key);
+          } catch (RuntimeException e) {
+            LOG.debug("Cannot get old value for the given key {}", key.getName());
+          }
+          if ("<UNSET_VALUE>".equals(entry.getValue())) {
```

Comment:
make "<UNSET_VALUE>" a constant

Reply:
I think you can use a magic value, but that should be internal and not exposed to the user at all (so we can change the magic value without telling all users). 

You can put this in `PropertyKey.UNSET_VALUE` as a constant. The `unsetConf` command just sends this `key=PropertyKey.UNSET_VALUE` and send a map `{key=PropertyKey.UNSET_VALUE}` to MetaMaster. WDYT?

Reply:
Abstract it to a constant.